bower.json

```diff
@@ -11,11 +11,11 @@
   "license": "Apache License, Version 2.0",
   "repository": {
     "type": "git",
-    "url": "git://github.com/mtscout6/react-router-bootstrap.git"
+    "url": "git://github.com/react-router/react-router-bootstrap.git"
   },
   "dependencies": {
-    "react-router": "^0.11.6",
-    "react-bootstrap": "^0.13.2"
+    "react-bootstrap": ">=0.15",
+    "react-router": ">=0.13.1"
```

Comment:
Makes sense, I forgot about that.

I'm not personally a bower user, what would you prefer? Your webpack/browserify/require process would be slimmer if it was a declared dependency. May be worth updating in react-bootstrap too then.